### DISCUSSION TITLE
help me with ```executiontime```

### DISCUSSION THREAD

**Discussion by AtifChy** [JanDeDobbeleer/oh-my-posh#3667] [General]
currently I have my configuration like this
![Screenshot 2023-04-03 003943](https://user-images.githubusercontent.com/42291930/229516086-cf77a719-7845-4736-975e-f4f1814c4917.png)

but i want to make it look like this 
![Screenshot 2023-04-03 004254](https://user-images.githubusercontent.com/42291930/229516161-62626218-948c-4b11-8f1a-873404b5ff3f.png)

how do I remove the output of milliseconds completely and remove the space between hour, minute,.. ?

here is the ```executiontime``` part of my config:
```{
          "type": "executiontime",
          "style": "plain",
          "foreground": "white",
          "template": "took <b><yellow>{{ .FormattedMs }}</> ",
          "properties": {
            "style": "austin",
            "threshold": 5000
          }
      }
```

**Comment by JanDeDobbeleer** [comment]
You're looking for `round` and can actually trim the spaces using built-in functions:

```json
{
  "type": "executiontime",
  "style": "plain",
  "foreground": "white",
  "template": "took <b><yellow>{{ nospace .FormattedMs }}</> ",
  "properties": {
    "style": "round",
    "threshold": 1000
  }
}
```

**Reply by AtifChy** [reply]
Thanks
